src/odemis/gui/cont/settings.py

```diff
@@ -832,6 +866,42 @@ def on_preparation(self, is_preparing):
     def enable(self, enabled):
         self.panel_center.Enable(enabled)
 
+# class EKSettingsController(SettingsBarController):
+#     """
+#     Controller, which provides the user with the option to
+#     select among different configurations regarding the mirror position. For example, the user can select the
```

Comment:
Can you add a comment on why this is commented out? (Or maybe that is added in another commit)